README.md

```diff
@@ -10,6 +10,20 @@ A collection of open source samples that illustrate best practices for
 The easiest way to browse through the samples in this repo (as well as a few others!)
 is the [visual samples index](https://flutter.github.io/samples).
 
+## Tip: download a single example using svn
```

Comment:
Now that this tip isn't just about using `svn`, can we re-word the tip summary to reflect this? e.g. Tip: reducing the size of your checkout

Thoughts?

Reply:
That's a good point. 

Something like "Tip: minimize download size" would cover both.

@noamraph, what do you think?